CHANGELOG.md

```diff
@@ -24,10 +24,11 @@ requesting a new account and will be displayed in the User Accounts module (PR #
 - Candidate's age can be retrieved from the Candidate class in days, months, or years (PR #5945)
 - Addition of autoSelect prop to React SelectElement allows for auto-selection of only available select option (PR #6156)
 - An `AcquisitionDate` field has been added to the `files` table (PR #6892)
-- The default value of the ScannerID field of the mri_protocol table is now NULL instead of 0 (PR #7496).
```

Comment:
why is this deleted?

Reply:
good catch! Fixed in the next commit. Probably inserted after rebase...